src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,45 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private K[] keys;
+    private V[] values;
+    private int size;
+
+    public StorageImpl() {
+        keys = (K[]) new Object[MAX_SIZE];
+        values = (V[]) new Object[MAX_SIZE];
+    }
+
+    public int indexOf(K key) {
```

Comment:
```suggestion
    private int indexOf(K key) {
```

should be placed after all public methods